app/src/main/kotlin/com/wire/android/ui/conversation/Conversation.kt

```diff
@@ -52,30 +61,59 @@ private fun ConversationListFloatingActionButton() {
 }
 
 @Composable
-private fun ConversationList(conversations: List<Conversation>) {
-    LazyColumn(
-        verticalArrangement = Arrangement.spacedBy(1.dp),
-    ) {
-        items(conversations) { conversation ->
-            ConversationRow(conversation)
+private fun Conversations(conversations: Map<ConversationFolder, List<Conversation>>) {
+    LazyColumn {
+        conversations.forEach { (conversationFolder, conversationList) ->
+            item { ConversationFolderHeader(name = conversationFolder.folderName) }
+            items(conversationList) { conversation ->
+                Box(modifier = Modifier.padding(1.dp)) {
+                    ConversationRow(conversation)
+                }
+            }
         }
     }
```

Comment:
Is it this easy to have multiple item types and folders in lists using Compose? WOW 🤯

Reply:
I guess for now it works :P Hopefuly it will be that easy in the future